codegen/src/getCli.ts

```diff
@@ -8,21 +8,13 @@ import { Ora } from 'ora';
 import { createReadStream, createWriteStream } from 'fs';
 
 import { cliPath } from './cliPath';
-import chalk from 'chalk';
 import { getCliPlatformFromNodePlatform } from './getCliPlatformFromNodePlatform';
 
 type GitHubResponse = {
   assets: { browser_download_url: string }[];
 };
 
 export const getCli = async ({ spinner }: { spinner: Ora }) => {
-  console.log();
-  spinner.info(
-    `You don't have the Xata CLI installed on your system, so we're downloading it. You could make this process faster by installing the CLI locally. More info: ${chalk.blueBright(
-      'https://docs.xata.io/cli/getting-started'
-    )}.
```

Comment:
Can we still show a message if 1) CLI not found installing it or 2) CLI outdated, updating it?

Reply:
Updated. 🙌